### PR TITLE
Bump version 4.2.0

### PR DESCRIPTION
**Pull request by planktonic**
## Changes

- Updates version number & changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 - `u-height` utility class is now configurable using `$bitstyles-height-values` and `$bitstyles-height-breakpoints`
 - New `u-min-width` utility class
+- New atom component `a-skip-link`, for skip-to-content links
 
 ### Breaking
 
 - `u-round` classes have been renamed `u-border-radius`
 
-## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2021-12-20
+## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2022-22-02
```
Comment (iampeternguyen) [inline CHANGELOG.md:17]: ? I find the year-day-month format a bit weird.

```suggestion
## [[4.1.0]](https://github.com/bitcrowd/bitstyles/releases/tag/v4.1.0) - 2022-02-22
```

**Review by maltoe — APPROVED** [review]
🎉

File: CHANGELOG.md (hunk 1 above)
Comment (iampeternguyen) [reply CHANGELOG.md:17]: oh looks like above is also y-m-d format.

**Review by iampeternguyen — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (planktonic) [reply CHANGELOG.md]: That was reverse US format apparently 🙈 definitely not what we want. Thanks!

**Review by ArinzeJeffrey-droid — APPROVED** [review]

**Pull request merged**